src/containers/barray.mli

```diff
@@ -64,10 +64,10 @@ val fill : 'a array -> int -> int -> 'a -> unit
     Raise [Invalid_argument "Array.fill"] if [ofs] and [len] do not
     designate a valid subarray of [a]. *)
 
-val to_list : 'a array -> 'a list
+val toList : 'a array -> 'a list
 (** [Array.to_list a] returns the list of all the elements of [a]. *)
 
-val of_list : 'a list -> 'a array
+val fromList : 'a list -> 'a array
```

Comment:
Personally I'd recommend not including functions that can't be implemented efficiently.